src/main/js/controls/Bubble/helpers/helpers.js

```diff
@@ -0,0 +1,559 @@
+ *
+ * @private
+ * @param {Array} nodes - List of d3.selection objects that are circles form bubble graph
+ * @returns {undefined} - returns nothing
+ */
+const enforceBubbleBlur = (nodes) =>
+    nodes.forEach((ele) => {
+        const node = d3.select(ele);
+        if (node.attr("aria-selected") === "false") {
+            node.classed(styles.bubbleBlur, true);
+        }
+    });
+
+/**
+ * Removes the carbon-bubbleBlur style from all the bubbles to unblur all the bubbles in the bubble graph.
+ *
+ * @private
+ * @param {Array} nodes - List of d3.selection objects that are circles in the bubbles graph.
+ * @returns {undefined} - returns nothing
+ */
+const removeBubbleBlur = (nodes) =>
+    nodes.forEach((ele) => {
+        const node = d3.select(ele);
+        if (node.attr("aria-selected") === "false") {
+            node.classed(styles.bubbleBlur, false);
+        } else {
+            node.attr("aria-selected", false);
+        }
+    });
+
+/**
+ * Handler for the bubble that is hovered on. It blurs all other bubble in the bubble graph except one which is selected.
+ *
+ * @private
+ * @param {HTMLElement} target - Target element bubble hovered on
+ * @param {string} hoverState - Mouse over or Mouse out
+ * @returns {undefined} - returns nothing
+ */
+const blurActionHandler = (target, hoverState) => {
+    d3.select(target.firstChild).attr("aria-selected", true);
+    const allCircleNodes = getAllCircleNodes();
+    if (hoverState === constants.HOVER_EVENT.MOUSE_ENTER) {
+        enforceBubbleBlur(allCircleNodes);
+    }
+};
+
+/**
+ * Draws the points with options opted in the input JSON by the consumer for each data set.
+ *  Render the point with appropriate color, x and y co-ordinates, label etc.
+ *  On click content callback function is called.
+ *
+ * @private
+ * @param {object} scale - d3 scale for Graph
+ * @param {object} config - Graph config object derived from input JSON
+ * @param {Array} pointGroupPath - d3 html element of the points group
+ * @param {object} dataTarget - data for the bubble graph
+ * @returns {undefined} - returns nothing
+ */
+const drawBubbles = (scale, config, pointGroupPath, dataTarget) => {
+    const renderDataPoint = (path, value, index) => {
+        path.append("g")
+            .classed(styles.point, true)
+            .attr("aria-disabled", !utils.isDefined(value.onClick))
+            .attr("transform", transformPoint(scale)(value))
+            .attr("aria-describedby", `${value.key}`)
+            .attr("aria-hidden", (value) =>
+                shouldHideDataPoints(config.shownTargets, value)
+            )
+            .on("click", function() {
+                dataPointActionHandler(value, index, this);
+            })
+            .append("circle")
+            .attr("aria-describedby", value.key)
+            .attr("r", (d) => decideRadius(dataTarget, d))
+            .attr("aria-selected", false)
+            .style("fill", (d) => decideColor(dataTarget, d))
```

Comment:
Instead of `style` you can use `attr("fill", ...)` like the rest of them

Reply:
yes, I will make the change